src/Error/Debugger.php

```diff
@@ -110,6 +113,23 @@ class Debugger
         ],
     ];
 
+    /**
+     * Mapping for error renderers.
+     *
+     * Error renderers are replacing output formatting with
+     * an object based system. Having Debugger handle and render errors
+     * will be deprecated and the new ErrorTrap system should be used instead.
+     *
+     * @var array<string, class-string>
+     */
+    protected $renderers = [
+        // Backwards compatible alias for text that will be deprecated.
+        'txt' => TextRenderer::class,
+        'text' => TextRenderer::class,
+        // The html alias currently uses no JS and will be deprecated.
+        'js' => HtmlRenderer::class,
```

Comment:
Do we need a `json` renderer?

Reply:
Debugger doesn't currently have a JSON output format.